Error while checking build/O2/fullCI for 4989b22bc08968718fe3dcced30e2a8fe8a79311 at 2023-04-25 00:08:
```
## sw/BUILD/O2-full-system-test-latest/log
Detected critical problem in logfile qedsim.log
[ERROR] SHUTTING DOWN DUE TO SIGNALED EXIT IN COMPONENT 204
Error in <TGeoCompositeShape::MakeNode>: Expression has no boolean operation
Error in <TGeoCompositeShape::ctor>: Composite halfFixationToPipe: cannot parse expression: pipeSupportTubeCarbon


## sw/BUILD/o2checkcode-latest/log
--
++ echo '========== List of errors found =========='
========== List of errors found ==========
++ GRERR=0
++ grep -v clang-diagnostic-error error-log.txt
++ grep ' error:'
++ GRERR=1
++ [[ 1 == 0 ]]
++ mkdir -p /sw/INSTALLROOT/599c9902dcf001cce1b8e34d0dd8584f86514272/slc8_x86-64/o2checkcode/1.0-local41/etc/modulefiles
++ cat
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/11224/4989b22bc08968718fe3dcced30e2a8fe8a79311/build_O2_fullCI/pretty.html).

--- a/Detectors/ITSMFT/ITS/simulation/src/Detector.cxx
+++ b/Detectors/ITSMFT/ITS/simulation/src/Detector.cxx
@@ -638,8 +638,8 @@ void Detector::createMaterials()
   o2::base::Detector::Medium(41, "C4SERVICES$", 41, 0, ifield, fieldm, tmaxfdSi, stemaxSi, deemaxSi, epsilSi, stminSi);
 
   // Polymer for ITS services
-  o2::base::Detector::Mixture(42, "POLY4SERVICES", aPoly, zPoly, dPoly, 2, wPoly);
-  o2::base::Detector::Medium(42, "POLY4SERVICES", 42, 0, ifield, fieldm, tmaxfdSi, stemaxSi, deemaxSi, epsilSi, stminSi);
+  o2::base::Detector::Mixture(42, "POLY4SERVICES$", aPoly, zPoly, dPoly, 2, wPoly);
+  o2::base::Detector::Medium(42, "POLY4SERVICES$", 42, 0, ifield, fieldm, tmaxfdSi, stemaxSi, deemaxSi, epsilSi, stminSi);
 
   // For ITS3
 

